builders/integrations/oracles/api3.md

```diff
@@ -119,32 +119,32 @@ The `_rrpAddress` is the main `airnodeRrpAddress`. The RRP contracts have alread
 
 ### Response Parameters {: #response-params }
 
 The callback to the requester contract contains two parameters:
 
-- [**`requestId`**](https://docs.api3.org/reference/airnode/latest/concepts/request.html#requestid){target=\_blank} - first acquired when making the request and passed here as a reference to identify the request for which the response is intended
+- [**`requestId`**](https://airnode-docs.api3.org/reference/airnode/latest/concepts/request.html#requestid){target=\_blank} - first acquired when making the request and passed here as a reference to identify the request for which the response is intended
 - **`data`** - in case of a successful response, this is the requested data encoded and contains a timestamp in addition to other response data. Decode it using the `decode()` function from the `abi` object
 
 !!! note
     Sponsors should not fund a `sponsorWallet` with more than they can trust the Airnode with, as the Airnode controls the private key to the `sponsorWallet`. The deployer of such Airnode undertakes no custody obligations, and the risk of loss or misuse of any excess funds sent to the `sponsorWallet` remains with the sponsor.
 
 ## dAPIs: API3 Data Feeds {: #dapis }
 
-[dAPIs](https://docs.api3.org/oev-searchers/in-depth/dapis/#dapis){target=\_blank} are continuously updated streams of off-chain data, such as the latest cryptocurrency, stock, and commodity prices. They can power decentralized applications such as DeFi lending, synthetic assets, stablecoins, derivatives, NFTs, and more.
+[dAPIs](https://airnode-docs.api3.org/reference/airnode/latest/concepts/airnode.html){target=\_blank} are continuously updated streams of off-chain data, such as the latest cryptocurrency, stock, and commodity prices. They can power decentralized applications such as DeFi lending, synthetic assets, stablecoins, derivatives, NFTs, and more.
 
 The data feeds are continuously updated by [first-party oracles](https://docs.api3.org/oev-searchers/glossary.html#first-party-oracles){target=\_blank} using signed data. DApp owners can read the on-chain value of any dAPI in real-time.
 
 Because they are composed of first-party data feeds, dAPIs offer security, transparency, cost-efficiency, and scalability in a turnkey package.
 
 ![The API3 Market dashboard.](/images/builders/integrations/oracles/api3/api3-3.webp)
 
-To learn more about how dAPIs work, please refer to [API3's documentation](https://docs.api3.org/oev-searchers/in-depth/dapis/#dapis){target=\_blank}.
+To learn more about how dAPIs work, please refer to [API3's documentation](https://docs.api3.org/oev-searchers/glossary.html#dapi){target=\_blank}.
```

Comment:
This page provides more info than the glossary: https://docs.api3.org/oev-searchers/in-depth/data-feeds/